CONTRIBUTING.md

```diff
@@ -15,6 +15,54 @@ Relevant coding style guidelines are the Go Code Review Comments and the Formatt
 
 In order to contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
 
+## Issues & Pull Requests
+
+### Creating an Issue
+
+Before **creating** an Issue i.e for `features`/`bugs`/`improvements` please follow these steps:
+
+
+1. Search existing Issues before creating a new Issue (look to see if the Issue has already been created).
+1. If it doesn't exist create a new Issue giving as much context as possible (please take note and select the correct Issue type, for example `bug`, `documentation` or `feature`.
```

Comment:
Please rephrase by removing the repeated words such as search, see, look.
It would be great if words like issue , pull requests to be in lowercase when they are not at start of the sentence.